package/Dockerfile

```diff
@@ -2,15 +2,17 @@ FROM ubuntu:18.04
 
 ARG ARCH=amd64
 
-RUN apt-get update && apt-get install -y kmod curl nfs-common fuse \
-        libibverbs1 librdmacm1 libconfig-general-perl libaio1 sg3-utils \
-        iputils-ping telnet iperf qemu-utils wget iproute2
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y kmod curl nfs-common fuse \
+    libibverbs1 librdmacm1 libconfig-general-perl libaio1 sg3-utils \
+    iputils-ping telnet iperf qemu-utils wget iproute2
 
 # Install grpc_health_probe
 RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-${ARCH} -O /usr/local/bin/grpc_health_probe && \
     chmod +x /usr/local/bin/grpc_health_probe
 
-COPY bin/longhorn bin/longhorn-instance-manager /usr/local/bin/
```

Comment:
This broke `launch-simple-longhorn` and our recommended way of exporting a volume from a single replica. https://longhorn.io/docs/1.1.2/advanced-resources/data-recovery/export-from-replica/